docs/release-information/update-to-version-3-200/update-platform-and-modules-to-3-200.md

```diff
@@ -1,34 +1,35 @@
-# Update platform and modules to .NET 6 version
+# Updating Platform and Modules to .NET 6
 
 ## Introduction
 
-This article describes how to update VC Platform manager and VC modules to .NET 6 (version since 3.2xx)
+This document will guide you through updating VC Platform Manager and VC modules (version 3.2xx or higher) to .NET 6.
 
-## Before the update
-Please review the [.NET 6 release notes](https://github.com/dotnet/core/blob/main/release-notes/6.0/README.md).
+## Prior to Update
+Before you start running your update, please review [.NET 6 Release Notes](https://github.com/dotnet/core/blob/main/release-notes/6.0/README.md).
 
-### Breaking change note
-Because of breaking changes introduced in the Entity Framework and for some other reasons, VC modules and platform are imcompatible with different versions of each other.
-VC Platform's minor version **equal or above 3.2xx** can't load and manage VC modules version **below 3.2xx**. And the other way round.
-For the described before reasons there is no chance for partial update. **You need to update to .NET 6 all your project.**
+### Breaking Change Note
+Because of some breaking changes introduced into the Entity Framework, as well as for some other reasons, VC modules are imcompatible with the Platform having a different version, and vice versa.
+Technically, VC Platform **version 3.2xx or higher** cannot load and manage VC modules with any version **below 3.2xx**, and the other way round.
+For the above reasons, there is no option for partial update, which means **you have to update your entire project to .NET 6.**
 
-### Slow migration
-Order module particularly contains a potentially long perform migration which could be in case your project have a lot of order's. Consider to apply it manually.
+### Slow Performance
+For the Order module, it may take particularly long to migrate in case your project has a lot of orders. For this reason, you might want to do the migration manually.
```

Comment:
Probably better to avoid this specific case and add link offline db migration document